samples/AngelScript/payload.as

```diff
@@ -0,0 +1,396 @@
+// Sample script.
```

Comment:
If, in AngelScript, `.as` and `.acs` file have exactly the same meaning and syntax, would it be possible to rename this file to `payload.acs`?

I'm worried we might have a conflict on `.acs` in the future, in which case we'll have to find a `.acs` sample file for AngelScript. We might as well take care of this now.

Reply:
Quite honestly, I didn't modify the extensions part to honor @ifarbod's initial PR. Personally I've never seen the `.acs` or `.angelscript` file extensions being used. In fact, all samples in the Angelscript SDK are using `.as` for file extensions, and I can't find any reference to "acs" in the Angelscript documentation.

Personally I'd lean more towards removing those 2 file extensions altogether, but I'd like to hear from @ifarbod first! 😊 (If not `.angelscript`, then only `.acs`, since that seems like the strangest file extension to me.)

Reply:
I've talked to AngelScript's author regarding the extension situation before, asking them for alternatives to `.as` where it's not possible, They said it's fine to use either `.acs` or `.angelscript`. They even initially wanted to use `.acs` as the "official" file extension for it. But if an extension's going to be removed, I'd say remove `.acs`, but keep support for `.angelscript` as the usage for this extension is more common than `.acs`.

Cheers

Reply:
You're right, searching github for `.angelscript` brings up a lot of stuff while searching for `.acs` brings up a lot of unrelated stuff (some unrelated Doom scripting language as well as XML-formatted files.) I guess I'll remove the `.acs` extension.

Reply:
I think It's all about legacy. AngelScript built its legacy on `.as` and `.angelscript` file extensions. 😉